### PR TITLE
Awet: Completed Dom-ajax exercises

### PR DESCRIPTION
**Pull request by afessha**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -60,45 +60,45 @@
...
       <div class="my-auto">
-        <h1 class="mb-0">Lisa
-          <span class="text-primary">Simpson</span>
+        <h1 class="mb-0">Awet
+          <span class="text-primary">Fessha</span>
         </h1>
-        <div class="subheading mb-5">Glasgow · G1 (+44) 758-777-777 ·
-          <a href="mailto:name@email.com">name@email.com</a>
+        <div class="subheading mb-5">Glasgow · G1 (+44) 745-494-1123 ·
+          <a href="mailto:name@email.com">awetkifle94@gmail.com</a>
```
Comment (tekul) [inline index.html]: Don't forget to change the `mailto` link too.

File: js/pull-requests.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    mySearch.addEventListener("keyup", function(event) {
+      const value = event.target.value;
+      pullRequestList.innerHTML = "";
+      pullRequests.forEach(function(pullRequest) {
+        // console.log(pullRequest.user.login);
+        // console.log(value);
+        //  console.log(pullRequest.user.login.search(value));
+
+        if (
+          pullRequest.user.login.toLowerCase().search(value.toLowerCase()) !==
```
Comment (tekul) [inline js/pull-requests.js]: You can use the [`includes` method](https://www.w3schools.com/jsref/jsref_includes.asp) to check whether a string contains another one. An alternative would be `startsWith` if you want to match from the beginning of the string.

File: index.html [hunk 3]
```diff
@@ -29,22 +29,22 @@
...
+      <span class="navbar-toggler-icon"></span>
+    </button>
     <div class="collapse navbar-collapse" id="navbarSupportedContent">
       <ul class="navbar-nav">
         <li class="nav-item">
           <a class="nav-link js-scroll-trigger" href="index.html">About me</a>
         </li>
         <li class="nav-item">
-          <a class="nav-link js-scroll-trigger" href="pages/my-github.html">My Github</a>
+          <a id="gitHubBtn" class="nav-link js-scroll-trigger" href="pages/my-github.html">My Github</a>
```
Comment (tekul) [inline index.html:47]: What is the `id` for here (and in the other changes below)?

**Review by tekul — APPROVED** [review]
This looks really good Awet. Just made a couple of comments, plus I wasn't sure what the `gitHubBtn` id was intended for. Excellent work :100: .

File: index.html (hunk 1 above)
Comment (afessha) [reply index.html]: Thanks for your feedback @tekul i will change it just now.

File: index.html (hunk 3 above)
Comment (afessha) [reply index.html:47]: it's just i tried to select the link using an id at first,  then I realised that I don't need it,  but forgot to delete it. i will remove it just now.

**Pull request closed without merging**